index.js

```diff
@@ -84,6 +142,19 @@ router.get('/pointers/:name', async ctx => {
   }
 })
 
+// Delete revshare pointer
+router.del('/pointers/:name', auth.authorize, async ctx => {
+  try {
+    const pointer = await db.get('pointer:' + ctx.params.name)
+    if (pointer) {
+      await db.del('pointer:' + ctx.params.name)
+      ctx.body = 'deleted pointer. name=' + ctx.params.name
```

Comment:
just for consistency, all these endpoints should return json. So this could even be something like `{ deleted: true, name: ctx.params.name }`